pep-0572.rst

```diff
@@ -201,6 +201,19 @@ in order to avoid ambiguities or user confusion:
   ungrouped assortment of symbols and operators composed of ``:`` and
   ``=`` is hard to read correctly.
 
+- Unparenthesized assignment expressions are prohibited in lambda functions.
+  Example::
+
+    (lambda: x := 1) # INVALID
+    lambda: (x := 1) # Valid, but unlikely to be useful
+    (x := lambda: 1) # Valid
+
+  This allows ``lambda`` to always bind less tightly than ``:=``; creating
+  a name binding inside the lambda function is unlikely to be of value, as
+  there is no way to make use of it. In cases where the name will be used
```

Comment:
"creating a name binding inside the lambda function" ->
"having a name binding at the top level inside a lambda function"

Reply:
Sure